### ISSUE TITLE
Images are not sent

### ISSUE THREAD

**Issue by jchiavellibr** [wppconnect-team/wa-js#1768]
### WA-JS Version

2.28.1

### WhatsApp Version

2.2412.50

### Browser Version

Brave 1.64.109 / Chromium 123.0.6312.58

### Operation System

Windows 11

### Steps to reproduce

Paste images from clipboard or upload from computer.

### What is expected?

Images must be sent when pasting or uploading from the computer

### What is actually happening?

When pasting images from the clipboard into a conversation or uploading from your computer, the images are not sent.
When you disable library injection, images are sent normally.
![image](https://github.com/wppconnect-team/wa-js/assets/39078199/1247e0a5-928f-4573-b1a8-c1e71e5524ed)

**Comment by icleitoncosta** [comment]
Hi, I tried to reproduce your error here, but without success. I used the same version, injected, logged in, pasted image, sent by selecting through the menu, sent via WPP, and no error was presented. Could you provide more details on how you are injecting, or if this error was a temporary one?

**Comment by jchiavellibr** [comment]
I discovered that the problem only happens in groups, and is not restricted to images, but to messages as well.

I'm using the library in an extension, and I inject it as follows:

[parasite.js]
...
async function load() {
     setTimeout(async ()=>{
         await import("./wppconnect-wa.js");
         await import("../main.js");
     } , 1000);
}
...

The library is loading normally:
![image](https://github.com/wppconnect-team/wa-js/assets/39078199/c5917adc-61fa-4d11-abed-fd4df92010db)


In the previous version of lib, the problem did not occur. To generate a new version of the extension, I just replaced the "wppconnect-wa.js" file with the new one (2.28.1) and recompiled the project.
